### DISCUSSION TITLE
When is the next Release - 0.85.0 ?

### DISCUSSION THREAD

**Discussion by thiencisco** [plouc/nivo#2526] [Q&A]
Hello friends,

I am waiting for the next release to come out - specifically 0.85.0.  I need the following commit to go into the release:

https://github.com/plouc/nivo/commit/c741a8832990b60c58e2e4041394e02f4388c88a

I have been waiting 3 months for the fix on the security vulnerability with d3-color 2.0.0.

It's detected from Snyk, it's public / general Security Vulnerabilities tracker:
https://security.snyk.io/vuln/SNYK-JS-D3COLOR-1076592

How can I get the release expedited?

Thanks,
Tim

**Comment by ankosasi** [comment]
+1 would really love this as well

**Reply by aashay94** [reply]
+1 waiting for this one to be released asap

**Comment by plouc** [comment]
There are a few pending PRs I'd like to include in the release, once merged, I'll try to release.

**Reply by thiencisco** [reply]
Thanks for the quick response.  Do you have the ETA of the release?  In days, weeks?

**Reply by plouc** [reply]
I think I should be able to release in a few days.
